Library/Homebrew/software_spec.rb

```diff
@@ -53,7 +53,7 @@ def owner=(owner)
     @resource.owner = self
     resources.each_value do |r|
       r.owner     = self
-      r.version ||= version
+      r.version ||= version.dup
```

Comment:
I mean what if we call `dup` on HEAD version, is there commit at the moment we `call` dup? So, `Version.create` and `version.dup` are almost the same things, however `Version.create` is easier to understand.